apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template-inserter.php

```diff
@@ -157,6 +166,72 @@ public function insert_default_template_data() {
 		add_option( $this->fse_template_data_option, true );
 	}
 
+	/**
+	 * Determines whether default pages have already been created.
+	 *
+	 * @return bool True if default pages have already been created, false otherwise.
+	 */
+	public function is_pages_data_inserted() {
+		return get_option( $this->fse_page_data_option ) ? true : false;
+	}
+
+	/**
+	 * Inserts default About and Contact pages based on Starter Page Templates content.
+	 *
+	 * The insertion will not happen if this data has been already inserted or if pages
+	 * with 'About' and 'Contact' titles already exist.
+	 */
+	public function insert_default_pages() {
+		// Bail if this data has already been inserted.
+		if ( $this->is_pages_data_inserted() ) {
+			return;
+		}
+
+		$request_url = 'https://public-api.wordpress.com/wpcom/v2/verticals/m1/templates';
+		$response    = wp_remote_get( $request_url );
+
+		if ( is_wp_error( $response ) ) {
+			return;
+		}
+
+		$api_response = json_decode( wp_remote_retrieve_body( $response ), true );
+
+		$about_page_content   = '';
+		$contact_page_content = '';
+
+		if ( ! empty( $api_response['templates'][6]['content'] ) ) {
+			$about_page_content = $api_response['templates'][6]['content'];
+		}
+
+		if ( ! empty( $api_response['templates'][1]['content'] ) ) {
+			$contact_page_content = $api_response['templates'][1]['content'];
+		}
+
+		if ( empty( get_page_by_title( 'About' ) ) ) {
+			wp_insert_post(
+				[
+					'post_title'   => 'About',
```

Comment:
How do you feel if we translated these titles?

Reply:
I think that would make a lot of sense. :D

Reply:
The templates are translated, so the titles should be as well.

Reply:
Added translations for both the titles and updated the template requests to pass the locale.